Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -63,19 +63,35 @@ final class CodecTests: XCTestCase {
     struct BasicStruct: Decodable, Equatable {
         let int: Int
         let string: String
+
+        public static func == (lhs: BasicStruct, rhs: BasicStruct) -> Bool {
```

Comment:
added so tests still pass in swift 4.0 without synthesized `Equatable`

Reply:
:+1:

Reply:
Is this the issue Jason was running into the other day?